uppercase_vowels/script-template.py

```diff
@@ -0,0 +1 @@
+def uppercase_vowels(text: str) -> str: ...
```

Comment:
should we include `from string import punctuation` here? I think it's best practice to use this and then we have clarity what punctuation chars to include

Reply:
@xettrisomeman shall we provide this import in the template or would that give away too much?

Reply:
no, I don't think it is necessary. Most will find a way to do(manually or using functions)

Reply:
agreed, thanks